.ci/run.sh

```diff
@@ -98,6 +98,14 @@ case "${CI_JOB}" in
 		echo "INFO: Running cloud hypervisor metrics tests"
 		sudo -E PATH="$PATH" ".ci/run_metrics_PR_ci.sh"
 		;;
+	"METRICS_EXPERIMENTAL")
```

Comment:
It takes about 40 minutes without CI setup.  And this is not testing anything a requirement for current kata default (virtiofsd + DAX). That is why I decided to keep it separated  it will be no required and will be a scheduled job to know the status of DAX(or other new configs ) over the time.

Reply:
@jcvenegas thanks for the explanation

Reply:
@jcvenegas for me it looks fine and the other fixes are minor (spelling and so on) thanks